packages/web/src/lib/core/auth/StitchAuth.ts

```diff
@@ -22,6 +21,14 @@ interface StitchAuth {
 
   loginWithCredential(credential: StitchCredential): Promise<StitchUser>;
 
+  loginWithRedirect(credential: StitchRedirectCredential);
```

Comment:
[nit] can this return a `never`?

Reply:
I tried that. `Never` requires you to throw from the function, which would be hacky here